classes/components/forms/site/PKPSiteConfigForm.php

```diff
@@ -73,5 +76,17 @@ public function __construct($action, $locales, $site)
             'size' => 'small',
             'value' => $site->getData('minPasswordLength'),
         ]));
+
+        // This parameter is only valuable in a multi-instances installation.
+        if ($numberOfJournals > 1) {
```

Comment:
I think this field should be always visible. It's not clear for me, as a user, that there's a hidden option. Also, it will allow the administrator to pre-configure everything properly, before creating the journals.